README.md

```diff
@@ -109,25 +109,50 @@ That's it! You now have a fully-functional The Spaghetti Detective server that y
 
 # Configure The Spaghetti Detective OctoPrint Plugin to use your own server
 
-Before you can configure The Spaghetti Detective OctoPrint Plugin to use your own server, you need add a printer to The Spaghetti Detective server you just built and obtain the secret token for that. To do so:
+Before you can configure The Spaghetti Detective OctoPrint Plugin to use your own server, you need add a printer to The Spaghetti Detective server you just built and obtain the 6-digit Verification Code for that. To do so:
 
 1. Pointing your browser to `http://your_server_ip:3334`.
 
-1. Log in as a user (you can just login with `root@example.com` but it's more secure to use a non-admin user).
+2. Log in as a user (you can just login with `root@example.com` but it's more secure to use a non-admin user).
```

Comment:
Using `1.` is markdown's way to say these are numbered bullets. They will be displayed as the correct sequence numbers. The advantage of not hard-coding the sequence number is we don't have to change all the bullets if we decide to insert a line.

Reply:
Ok, so I should make all of them one?